src/main/resources/interface-specification.yml

```diff
@@ -220,9 +220,14 @@ paths:
           schema:
             type: string
         - in: query
-          name: state
+          name: states
+          description: comma separated sequence of agreement states to filter the response with
           schema:
-            $ref: '#/components/schemas/AgreementState'
+            type: array
+            items:
+              $ref: '#/components/schemas/AgreementState'
+            default: [ ]
+          explode: false
```

Comment:
🤔 what is this?

Reply:
it allows to generate the value as a comma separated string

Reply:
like a sir 🧐